### ISSUE TITLE
A project description more than 255 characters causes project creation/updating to fail

### ISSUE THREAD

**Issue by erwa** [azkaban/azkaban#258]
If you try to create a project with a description longer than 255 characters, it fails with

```
ERROR: Insert project for existing project failed. <project_name>
```

If you try to update the description of an existing project to something longer than 255 characters, you get

```
Error marking project <project-name> as inactive
```

We should either return better error messages or restrict the description text boxes to 255 characters.

**Comment by rohitsm** [comment]
This report is rather old and this bug has since been since fixed (checked with version 3.68).

The description limit is now capped at 2048 which is the maximum length of the the `description` column in the `projects` table as shown below:

**Original DDL script:** https://github.com/azkaban/azkaban/blob/master/azkaban-db/src/main/sql/create.projects.sql

``` sql
mysql> describe projects;
+------------------+---------------+------+-----+---------+----------------+
| Field            | Type          | Null | Key | Default | Extra          |
+------------------+---------------+------+-----+---------+----------------+
| id               | int(11)       | NO   | PRI | NULL    | auto_increment |
| name             | varchar(64)   | NO   | MUL | NULL    |                |
| active           | tinyint(1)    | YES  |     | NULL    |                |
| modified_time    | bigint(20)    | NO   |     | NULL    |                |
| create_time      | bigint(20)    | NO   |     | NULL    |                |
| version          | int(11)       | YES  |     | NULL    |                |
| last_modified_by | varchar(64)   | NO   |     | NULL    |                |
| description      | varchar(2048) | YES  |     | NULL    |                |
| enc_type         | tinyint(4)    | YES  |     | NULL    |                |
| settings_blob    | longblob      | YES  |     | NULL    |                |
+------------------+---------------+------+-----+---------+----------------+
10 rows in set (0.00 sec)
```
Exceeding 2048 characters, the process fails with the following errors, which is far more descriptive than what the OP originally specified above. 

- **Project creation:** `Insert <NAME OF PROJECT> for existing project failed`
 E.g.: `Insert temp2 for existing project failed`

- **Update Description of existing project:** `Error update Description, project <NAME OF PROJECT>`
  E.g.: `Error update Description, project Temp`

This bug could be closed. cc: @HappyRay and @kunkun-tang

**Comment by erwa** [comment]
Thanks for the update, @rohitsm .  Going to close this issue.
